src/mixins/getLocalTitleMixin.js

```diff
@@ -0,0 +1,6 @@
+export const getLocalTitle = {
+  methods: {
+    // Iterates through the study titles and returns the first with a non-empty string
+    getFirstTitle (obj) { for (const key in obj) if (obj[key] !== '') return obj[key] }
+  }
```

Comment:
I would actually give priority to English if present. This is because the web interface is in English.

Reply:
Regardless of whether an English title is given or not, it's always the first item of the `studytitles` object. In case that changes though, I could do this:

```
getLocalText (obj) {
      if (obj.en !== '') return obj.en
      else for (const key in obj) if (obj[key] !== '') return obj[key]
    }
```

This assures that the English title is returned (if non-empty).